packages/analytics/jest.config.js

```diff
@@ -1,188 +1,15 @@
-  // modulePathIgnorePatterns: [],
-
-  // Activates notifications for test results
-  // notify: false,
-
-  // An enum that specifies notification mode. Requires { notify: true }
-  // notifyMode: "failure-change",
-
-  // A preset that is used as a base for Jest's configuration
-  // preset: undefined,
-
-  // Run tests from one or more projects
-  // projects: undefined,
-
-  // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
-
-  // Automatically reset mock state between every test
-  // resetMocks: false,
-
-  // Reset the module registry before running each individual test
-  // resetModules: false,
-
-  // A path to a custom resolver
-  // resolver: undefined,
-
-  // Automatically restore mock state between every test
-  // restoreMocks: false,
-
-  // The root directory that Jest should scan for tests and modules within
-  // rootDir: undefined,
-
-  // A list of paths to directories that Jest should use to search for files in
-  // roots: [
-  //   "<rootDir>"
-  // ],
-
-  // Allows you to use a custom runner instead of Jest's default test runner
-  // runner: "jest-runner",
-
-  // The paths to modules that run some code to configure or set up the testing environment before each test
-  // setupFiles: [],
-
-  // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  setupFilesAfterEnv: ['<rootDir>/config/jest/setupTests.ts'],
-
-  // A list of paths to snapshot serializer modules Jest should use for snapshot testing
-  // snapshotSerializers: [],
-
-  // The test environment that will be used for testing
-  // testEnvironment: "jest-environment-jsdom",
-
-  // Options that will be passed to the testEnvironment
-  // testEnvironmentOptions: {},
-
-  // Adds a location field to test results
-  // testLocationInResults: false,
-
-  // The glob patterns Jest uses to detect test files
-  testMatch: ['**/?(*.)+(spec|test).ts?(x)'],
-
-  // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  testPathIgnorePatterns: ['/node_modules/', '/lib/'],
-
-  // The regexp pattern or array of patterns that Jest uses to detect test files
-  // testRegex: [],
-
-  // This option allows the use of a custom results processor
-  // testResultsProcessor: undefined,
-
-  // This option allows use of a custom test runner
-  // testRunner: "jasmine2",
-
-  // This option sets the URL for the jsdom environment. It is reflected in properties such as location.href
-  // testURL: "http://localhost",
-
-  // Setting this value to "fake" allows the use of fake timers for functions such as "setTimeout"
-  // timers: "real",
-
-  // A map from regular expressions to paths to transformers
-  transform: {
-    '^.+\\.(js|ts|tsx)$': '../../node_modules/@swc/jest',
+  ...config,
+  displayName: require('./package.json').name,
+  coverageThreshold: {
```

Comment:
why not set a global common threshold?

Reply:
Because I don't wanna packages that have high coverage to drop.
So the base sets it to 95% and we adjust where it is not met.

Reply:
I think we can set a high value in here for all projects and improve from there instead of _trying to make the numbers look nice_.

How well or poorly tested the code will not change because of this value changed. Honestly this is a metric I never care about, I think that it does not reflect how well tested your code is, for instance you can have a snapshot test that runs all your code and you will 100% coverage, and maybe not business logic tested at all.

Reply:
I wasn't making any number look nice. 
I just set the current coverage levels as the threshold so we don't go below it.

The only common number that will make all packages pass is 0.

If the team doesn't care about coverage levels, I can just turn it off. Your call.

Reply:
don't make the threshold pass (actually not a required test), set _a reasonable number_ we wanna achieve

Reply:
the threshold has been going down because in the past we only reported coverage for the components package and now we are reporting for all package, and the coverage is really low in some of them

Reply:
Would 70% be reasonable? If not, what number would you suggest?

Reply:
I will go for 90 or something like that. Something I realise is that we are counting tabler icons lines of code and we should not.

Reply:
Getting back to this. 

I have updated all packages to use the threshold from the base jest config. However, the `yarn test` command will fail because multiple packages won't meet the threshold.

Example:

```
@cmpsr/cloud-storage:test:coverage: Jest: "global" coverage threshold for branches (95%) not met: 77.78%
```

You requested to not adjust the threshold per package ("don't make the threshold pass")
What would be the suggestion for CI checks to pass then?

Reply:
We can run the command like this to pass this flag --coverageThreshold {} to jest and disable the threshold.

You can test it locally by running this command.

`yarn test --  -- --coverageThreshold {}`


That way we can simply remove the flag whenever we are ready to enable it back again.

WDYT?